### ISSUE TITLE
in-home-streaming  v0.1 input lag

### ISSUE THREAD

**Issue by CommicKick69ing** [jakibaki/In-Home-Switching#12]
This is an great app but its input problem makes it unplayable

**Comment by jakibaki** [comment]
Input problem?  
We get around 0.1 seconds input-delay and others reported similar numbers.

Chances are good that your network isn't up to the task, some wifi-routers just don't play well with this.

**Comment by co5mo** [comment]
maybe you can get some inspiration also here... https://github.com/polygraphene/ALVR?
they have the streaming thing figured out

**Comment by CommicKick69ing** [comment]
I get an input lag of around 0.7

**Comment by jacob5jac** [comment]
my input lag is also pretty bad. along the video compression. could it be a resolution issue? my monitor is 3440x1440. could have some difficulties compressing that to 720p.

**Comment by jakibaki** [comment]
@jacob5jac yeah you'll want to set your display-resolution to 720p if you want good performance.

**Comment by jacob5jac** [comment]
@jakibaki just swapped monitors to a 1080p one (displaying at 720p), made it my main display and it seems to help a little bit. still unplayable for MHw standards. the artifacting didn't seem to improve at all.

I'll try again tomorrow with only 1 monitor plugged in and after a restart see if that improves at all. I'll also go buy a switch compatible USB to Gb ethernet adapter to eliminate my wifi being the issue.
